Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using Console = System.Console;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
+        public static void Demo()
+        {
+            bool moreEntries = true;
+
+            while (moreEntries)
+            {
+            
+                string convertedStringName = ConvertToString("Enter your name.");
+                Console.WriteLine($"Your converted string is \"{convertedStringName}\"");
+
+                int convertedIntAge = ConvertToInt("\nEnter your age, no '.'");
+                Console.WriteLine($"Your converted int is \"{convertedIntAge}\"");
+
+                float convertedFloatHeight = ConvertToFloat("\nEnter your height in meters. '.' is ok to use.");
+                Console.WriteLine($"Your converted float is \"{convertedFloatHeight}\"");
+
+                float convertedFloatWeight = ConvertToFloat("\nEnter your weight in kg. '.' is ok to use.");
+                Console.WriteLine($"Your converted float is \"{convertedFloatWeight}\"\n");
+
+                CalculateBmi(convertedFloatWeight, convertedFloatHeight);
+
+                moreEntries = AddAnotherPerson();
+            }
+
+
+        }
+
+        public static bool AddAnotherPerson()
+        {
+            bool moreEntries = true;
+
+            Console.WriteLine("\nWould you like to enter another person? " +
+                                                           "Press Y to enter another or any other button to exit.");
+
+            string checkAnswer = Console.ReadLine().ToLowerInvariant();
+
+            if (checkAnswer != "y")
+            {
+                Console.WriteLine("You chose not to enter another user. Thank you, goodbye.");
+                moreEntries = false;
+            }
+
+            return moreEntries;
+
+        }
+
+
+        public static float CalculateBmi(float weightInKg, float height)
+        {
+            float bmi = weightInKg / (height * height);
+
+            if((weightInKg <= 0) && (height <= 0))
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'Failed calculating BMI. Reason:' 4 times.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4871920)

Reply:
This makes sense- you repeat the same message 4 times. What if the message changes? Then you will make 4 changes.